### PR TITLE
Package audit

### PR DESCRIPTION
**Pull request by rugulous**
Removes packages we don't need:
- `jimp`
- `del`
- `serve-favicon` (we don't actually have one to serve!)

Move some dependencies to devDependencies:
- `chalk`
- `gulp`
- `gulp-rename`
- `prompt-sync`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,26 +21,23 @@
...
 		"express-handlebars": "^6.0.7",
 		"express-session": "^1.17.3",
-		"gulp": "^4.0.2",
-		"gulp-rename": "^2.0.0",
-		"jimp": "^0.16.2",
 		"joi": "^17.7.0",
 		"morgan": "^1.10.0",
-		"prompt-sync": "^4.2.0",
-		"sequelize": "^6.28.0",
-		"serve-favicon": "^2.5.0"
```
Comment (matt-fidd) [inline package.json:43]: I don't know where this got lost but we definitely used to use this - and we should add it back in as it seems like it's missing from the current source

Comment (rugulous) [reply package.json:43]: They should be in dev dependencies now as I thought they were only used for gulp n that?

Comment (matt-fidd) [reply package.json:43]: Sorry, yes my comment was only on 'serve-favicon' but it looks like github highlighted everything

Comment (rugulous) [reply package.json:43]: Ah gotcha, thanks GitHub 👀

If I'm honest, do we actually need it? We don't currently have a favicon and I'm fairly sure we can show one with just html?

**Review by matt-fidd — APPROVED** [review]

**Pull request merged**